### DISCUSSION TITLE
How to clear token after logout

### DISCUSSION THREAD

**Discussion by SelenaWenna** [soofstad/react-oauth2-pkce#152] [General]
I call logOut(), but token was cached.
Then I call login() and user login automatic with last credentials.
In my config autoLogin: false
How can I login users with unique credentials every time?

**Comment by soofstad** [comment]
Hi!
I might need some more information, but most likely, you have not configured "logoutEndpoint" correctly. Or the identity provider does not implement it properly.

What is being cached is the cookie the IP sets, not the tokens and state of this library, that is being cleared on "logOut".

If this does not solve your issue, let me know, and please provide the full AuthConfig object :slightly_smiling_face:

**Comment by SelenaWenna** [comment]
@soofstad Thanks! With logoutEndpoint it works!!!
